core/server/master/src/main/java/alluxio/master/meta/MetaMasterSync.java

```diff
@@ -56,32 +54,22 @@
   /** Last System.currentTimeMillis() timestamp when a heartbeat successfully completed. */
   private long mLastSuccessfulHeartbeatMs;
 
+  /** The ID of this standby master. */
+  private AtomicReference<Long> mMasterId = new AtomicReference<>(-1L);
```

Comment:
`final`

Reply:
what does `-1` mean here? Uninitialized?  I would create a constant `UNINITIALIZED_MASTER_ID = -1` rather than using the constant literal directly

Reply:
Changed, thanks!